### ISSUE TITLE
Allow to change order via drag'n'drop

### ISSUE THREAD

**Issue by TheZoker** [ndom91/briefkasten#46]
### Clear and concise description of the problem

It would be very nice, if we could change the order of the bookmarks via drag'n'drop. So I can put my most important things at the top of the list.

### Suggested solution

Add bookmark ordering via drag'n'drop

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ndom91** [comment]
Yeah fair enough. You mean in the "List"/"Table" view of the bookmarks themselves?

![image.png](https://volta.s3.fr-par.scw.cloud/image_3f36eef260.png)

**Comment by TheZoker** [comment]
Yes exactly 👍

**Comment by ndom91** [comment]
Hey folks, I've completely rewritten this project using SvelteKit. I've carried over almost all of the features from the original version, and so I'm going to close all existing issues.

If you're still interested in the project, I'd ask you to check out the latest verion (available at the same URL as always - https://briefkastenhq.com), and open a new issue if necessary 🙏
